migration/identity-migration/src/main/java/io/camunda/migration/identity/MigrationRunner.java

```diff
@@ -38,18 +42,37 @@ public void run() {
     final String command =
         args.containsOption("command") ? args.getOptionValues("command").getFirst() : "migrate";
     if (!asList("migrate", "status").contains(command)) {
-      if ("migrate".equals(command)) {
-        authorizationMigrationHandler.migrate();
-      }
       throw new IllegalArgumentException("Unknown command: " + command);
     }
 
-    // TODO: place holder to logic
-    System.out.println("Migration Logic");
+    if ("migrate".equals(command)) {
+      migrate();
+    }
+  }
+
+  private void migrate() {
+    while (true) {
```

Comment:
:question: Will there be any conditions that the migration should end gracefully and not end up in a loop here?